### PR TITLE
Figurine voicelines

### PR DESCRIPTION
**Pull request by FATFSAAM2**
## About the PR
added 7 new figurine voice lines
2 for the captain
1 for the hos (i also changed voice line two which had a typo in it)
2 for the warden
1 for detective and one for the secoff figurine 

## Why / Balance
its just to add a little bit of flavor to the figurines and to fix a typo

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: added 7 new figurine voice lines.
- fix: changed a hos figurine voice line to not include a typo.

### CODE REVIEW

**Comment by Unkn0wnGh0st333** [comment]
Some of them are neat and funny but not to sure about the "Gib the CMO!" for captain 🤔

**Comment by FATFSAAM2** [comment]
> Some of them are neat and funny but not to sure about the "Gib the CMO!" for captain 🤔

someone recommended it on the discord and i thought it was funny and i also didnt have any other ideas so i added it 
i might remove it? i dont know honestly we will see

**Comment by Unkn0wnGh0st333** [comment]
"How did I get hired? Yes." is really good

**Comment by FATFSAAM2** [comment]
> "How did I get hired? Yes." is really good

cracked

**Comment by Hreno** [comment]
`somewhere` and `no` should be capitalized.

**Comment by FATFSAAM2** [comment]
is there anything else that i possibly missed????

**Review by UbaserB — APPROVED** [review]

**Comment by Hreno** [comment]
image

**Comment by FATFSAAM2** [comment]
> image

Biggest change in ss14 history

**Comment by lzk228** [comment]
such a things shouldn't have cl, read guidelines for that

File: Resources/Locale/en-US/datasets/figurines.ftl [hunk 1]
```diff
@@ -30,16 +30,23 @@ figurines-musician-2 = Never gonna let you down!
 # figurines-boxer-
 
 figurines-captain-1 = Glory to NT!
+figurines-captain-2 = How did I get hired? Yes.
+figurines-captain-3 = The nuclear disk is secure. where? Somewhere.
```
Comment (slarticodefast) [inline Resources/Locale/en-US/datasets/figurines.ftl:34]:
```suggestion
figurines-captain-3 = The nuclear disk is secure. Where? Somewhere.
```

**Pull request merged**